docs/resources/volume.md

```diff
@@ -93,7 +93,7 @@ resource "ionoscloud_volume" "example" {
   user_data               = "foo"
 }
 
-resource "ionoscloud_volume" "example" {
```

Comment:
@cristiGuranIonos maybe we can add an example in the documentation for the hostname, but no problem if we leave it as it is, it is explained well enough in the docs.

Reply:
we'll leave it as it is for now. should be clear enough how it's used.